user-guide/Advanced_Functionality/Failover/Viewing_Failover_information/Viewing_the_current_Failover_DMA_status.md

```diff
@@ -65,5 +65,13 @@ This will open the *Status* dialog box, which displays a number of status parame
 
     “Failing” means that the database server is probably not running or that there is a configuration issue.
 
+- **Pcap libraries**: "Pcap not installed" or "Could not be detected"
```

Comment:
"Pcap libraries" is probably not the real name of this status parameter. Could you provide me with the actual name, @HansSolos?

Reply:
From Npcap documentation: "Npcap is the Nmap Project's packet capture (and sending) library for Microsoft Windows".
Proposal: **Packet capture libraries**: "Pcap not installed" or "Could not be detected"

Reply:
```suggestion
- **Packet capture libraries**: "Pcap not installed" or "Could not be detected"
```